### DISCUSSION TITLE
Remove lumen from Illuminance units

### DISCUSSION THREAD

**Discussion by epenet** [home-assistant/architecture#843] [Entity Models] [answered]
As per https://en.wikipedia.org/wiki/Candela_per_square_metre#See_also, lumen is a unit of `Luminous flux`, not a unit of `Illuminance`

`Luminous flux` is luminous energy per unit time (ie. light energy transmitted)
`Illuminance` is luminous flux incident on a surface (ie. light received by a surface)

See https://github.com/home-assistant/core/pull/83411 and https://github.com/home-assistant/core/pull/84366

Implementation PR:
- [ ] core https://github.com/home-assistant/core/pull/84386
- [ ] dev docs https://github.com/home-assistant/developers.home-assistant/pull/1592
- [ ] docs https://github.com/home-assistant/home-assistant.io/pull/25378

**Comment by frenck** [comment, marked as answer]
Agree, especially for its main usage (sensors); it makes not a lot of sense to measure lumen (besides, it isn't illuminance at that point, indeed).

**Comment by MartinHjelmare** [comment]
Only 4-5 integrations seem affected.
